src/FixedDataTableRoot.js

```diff
@@ -11,10 +11,10 @@
 
 'use strict';
 
-import FixedDataTableContainer from 'FixedDataTableContainer';
-import FixedDataTableCellDefault from 'FixedDataTableCellDefault';
-import FixedDataTableColumn from 'FixedDataTableColumn';
-import FixedDataTableColumnGroup from 'FixedDataTableColumnGroup';
+const FixedDataTableContainer = require('FixedDataTableContainer').default;
```

Comment:
For some reason, we need to use `module.exports`, otherwise I'm not able to get the site running properly (page crashes because our examples can't import modules properly)
Since we use `module.exports` down below, we can't use `import` here.

Reply:
Weird, did you make any progress in figuring out what this is?  Do you have steps to reproduce if @mycrobe or I have time to take a look?

Reply:
> Weird, did you make any progress in figuring out what this is?

Nope :( 

> Do you have steps to reproduce if @mycrobe or I have time to take a look?

To reproduce, checkout this branch, and just use `export default FixedDataTableRoot;` instead of `module.exports = FixedDataTableRoot;`

Reply:
> Weird, did you make any progress in figuring out what this is? Do you have steps to reproduce if @mycrobe or I have time to take a look?

Actually, I think I found the issue.
The examples use named imports
```js
import { Table, Column, Cell } from 'fixed-data-table-2';
```
This doesn't work for default export, since it's a single object, and [we can't destructure in the import statement](https://github.com/babel/babel-loader/issues/194#issuecomment-168579479).

----
The fix would be to either destructure the object after importing:
```js
import FixedDataTableComponents from 'fixed-data-table-2';
const { Table, Column, Cell } = FixedDataTableComponents;
```
or just use a regular export (without default)
```js
export {
  Cell: FixedDataTableCellDefault,
  Column: FixedDataTableColumn,
  ColumnGroup: FixedDataTableColumnGroup,
  Table: FixedDataTableContainer,
  version: '1.0.0-beta.17',
};
```
@wcjordan , @mycrobe , thoughts?

Reply:
Ahhh ya that makes sense.  I recommend going with a regular export (without default).